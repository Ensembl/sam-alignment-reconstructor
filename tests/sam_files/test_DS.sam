MS2007487-600V3:1:1103:08049:16258	163	man2b1_T2_11_32098232-32098503	1	70	141M3D128M7S	=	1	271	CTCAAGAAAATGCAGGTTTGCATAAAAGTACCTGTGAAGAGATCGGCAATAGGTGGAGTCAGGCTCTCACTGGCCCTCCAGAGCATTTCCATCTCTTTGGTCTTCATCCTGCGGTCCTTGTCCTGGTAGTCCAAGCGGCCAAAGAAGCCGTCATAGCCCATCTATAAACAAATCAGAGCAGAGCCATCAGCAACTTTAGATATTAAAATACAATTCATGATGCCAGATAACATGCCATGGCTACCTGAGCAAAGATGGAGGCGTGTTCTAAACCAA	CECFGGGGGGGGGGFFEFFEAFFGGGGGGGGFGDGGFFGGGGGGGGGDGFFGDFGGGGGGGGCGF?EGGGGGGGFCFFGF=EFFGFFCGGGGGGFFG9FFEFG?F9;FEE=FFEEGGGCFEGGFFGFFGG;CE98EDEG8E8DDFFGGGDEEGFG7@FDGGGCGFGGCFGGGF**397;9*CCCFGAFFFF9FFF8@>=6F>FCFFF8C:+>F:C6<CFFE>EFE;@834=;<@E@E<;;A@E@1<<;;@E3;)/;E+*2*/:**;*<CCG9,,**	PG:Z:novoalign	AS:i:151	UQ:i:151	NM:i:3	MD:Z:141^A0^A0^G128	PQ:i:284	SM:i:70	AM:i:70	BC:Z:AAAAAA
