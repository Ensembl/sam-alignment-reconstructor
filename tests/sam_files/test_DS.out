MS2007487-600V3:1:1103:08049:16258
CTCAAGAAAATGCAGGTTTGCATAAAAGTACCTGTGAAGAGATCGGCAAT
||||||||||||||||||||||||||||||||||||||||||||||||||
CTCAAGAAAATGCAGGTTTGCATAAAAGTACCTGTGAAGAGATCGGCAAT

AGGTGGAGTCAGGCTCTCACTGGCCCTCCAGAGCATTTCCATCTCTTTGG
||||||||||||||||||||||||||||||||||||||||||||||||||
AGGTGGAGTCAGGCTCTCACTGGCCCTCCAGAGCATTTCCATCTCTTTGG

TCTTCATCCTGCGGTCCTTGTCCTGGTAGTCCAAGCGGCCAAAGAAGAAG
|||||||||||||||||||||||||||||||||||||||||   ||||||
TCTTCATCCTGCGGTCCTTGTCCTGGTAGTCCAAGCGGCCA---AAGAAG

CCGTCATAGCCCATCTATAAACAAATCAGAGCAGAGCCATCAGCAACTTT
||||||||||||||||||||||||||||||||||||||||||||||||||
CCGTCATAGCCCATCTATAAACAAATCAGAGCAGAGCCATCAGCAACTTT

AGATATTAAAATACAATTCATGATGCCAGATAACATGCCATGGCTACCTG
||||||||||||||||||||||||||||||||||||||||||||||||||
AGATATTAAAATACAATTCATGATGCCAGATAACATGCCATGGCTACCTG

AGCAAAGATGGAGGCGTGTTCT.......
||||||||||||||||||||||       
AGCAAAGATGGAGGCGTGTTCTaaaccaa

